lib/matplotlib/tests/test_collections.py

```diff
@@ -696,7 +696,7 @@ def test_pathcollection_legend_elements():
 
     h, l = sc.legend_elements(fmt="{x:g}")
     assert len(h) == 5
-    assert_array_equal(np.array(l).astype(float), np.arange(5))
+    assert_array_equal(l, ["0", "1", "2", "3", "4"])
```

Comment:
Is this a `list` of `str`? Can it be a regular `assert`?

Reply:
Indeed; switched more places to use plain asserts.